### PR TITLE
[PATCH wfy] Update url

### PR DESCRIPTION
**Pull request by damassi**
https://artsyproduct.atlassian.net/browse/DISCO-360

Related: https://github.com/artsy/emission/pull/1200

### CODE REVIEW

File: src/Components/Artwork/Details.tsx [hunk 1]
```diff
@@ -141,8 +142,15 @@ export class Details extends React.Component<Props, null> {
...
-      return sa.highest_bid.display || sa.opening_bid.display
+      const highestBidDisplay = get(
+        artwork,
+        p => p.sale_artwork.highest_bid.display
+      )
+      const openingBidDisplay = get(
+        artwork,
+        p => p.sale_artwork.opening_bid.display
+      )
+      return highestBidDisplay || openingBidDisplay || ""
```
Comment (damassi) [inline src/Components/Artwork/Details.tsx]: Fixes https://sentry.io/artsynet/force-production/issues/731894886/

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit ef585c07e74eeba6df1c6d5bda091bf820e5b952

https://deploy-preview-1421--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 5a6360d309c2d6c64523a2dc56af042b83ddb505

https://deploy-preview-1421--artsy-reaction.netlify.com

File: src/Components/Artwork/Details.tsx (hunk 1 above)
Comment (mzikherman) [reply src/Components/Artwork/Details.tsx]: since it's a regression it could use a test.

**Pull request merged**